frontend/src/metabase/lib/redux/hooks.unit.spec.tsx

```diff
@@ -52,7 +52,7 @@ describe("useDispatch", () => {
       const didNotFindEmailState = jest.fn();
 
       setup({
-        thunk: () => (dispatch: any, getState: () => State) => {
+        thunk: () => (_dispatch: any, getState: () => State) => {
```

Comment:
Just rename to match the convention even though this doesn't make ESLint error because `"args": "after-used"`